### PR TITLE
setup code for data preprocessing with Phylomizer and DarkHorse parsing

### PR DESCRIPTION
**Pull request by ekopylova**
Updates:
1. Install instructions for Phylomizer dependencies (will need to be added to conda-recipes later)
2. Update INSTALL.md to include OrthoFinder and Phylomizer
3. Add `launch_benchmark.sh` script which will preprocess genomes (e.g. run DIAMOND, shear species tree & build HMM models)
4. Add dummy function `parse_darkhorse()` to `parse_output.py` for Carly to work on
5. Add `utils.sh` for functions used by multiple shell scripts

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,36 @@ before_install:
...
+  - wget http://www.drive5.com/muscle/downloads3.8.31/muscle3.8.31_i86linux64.tar.gz
+  - tar zxvf muscle3.8.31_i86linux64.tar.gz
+  - mkdir muscle
+  - mv muscle3.8.31_i86linux64 muscle/muscle
+  - export PATH=$PWD/muscle:$PATH
+  # Install T-COFFEE
+  - wget http://www.tcoffee.org/Packages/Stable/Latest/linux/T-COFFEE_installer_Version_11.00.8cbe486_linux_x64.tar.gz
+  - tar zxvf T-COFFEE_installer_Version_11.00.8cbe486_linux_x64.tar.gz
+  - export PATH=$PWD/T-COFFEE_installer_Version_11.00.8cbe486_linux_x64/bin:$PATH
+  # INSTALL TRIMAL
```
Comment (antgonza) [inline .travis.yml]: Install? Obviously, not required.

File: .travis.yml [hunk 2]
```diff
@@ -13,6 +13,36 @@ before_install:
...
+  - wget https://github.com/scapella/trimal/archive/v1.4.1.tar.gz
+  - tar zxvf v1.4.1.tar.gz
+  - cd trimal-1.4.1/source
+  - make
+  - export PATH=$PWD:$PATH
+  - cd ../..
+  # INSTALL PHYML
```
Comment (antgonza) [inline .travis.yml]: Same here ...

File: benchmark/launch_benchmark.sh [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+# query species protein coding sequences in FASTA format
+query_species_coding_seqs_fp=$6
+# reference species protein coding sequences in FASTA format
+ref_species_coding_seqs_fp=$7
+# gene trees in Newick format
+gene_tree_dir=$8
+# gene multiple sequence alignment dir
+gene_msa_dir=$9
+# tabular DIAMOND alignments of query genome
+diamond_tabular_query_fp=${10}
```
Comment (antgonza) [inline benchmark/launch_benchmark.sh:35]: Why these are between {} but all <10 are not?

File: benchmark/launch_benchmark.sh [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+phylonet_install_dir=${11}
+# Jane 4 install dir
+jane_install_dir=${12}
+# T-REX install dir
+trex_install_dir=${13}
+# Verbose string 'true' or 'false'
+verbose=${14}
+# Initial command that precedes call to software
+# (example choosing virtualenv to workon)
+init_command="${15}"
```
Comment (antgonza) [inline benchmark/launch_benchmark.sh:46]: Does it need the "?

File: benchmark/parse_output.py [hunk 5]
```diff
@@ -89,6 +89,35 @@ def parse_consel(input_f):
     return pvalues
 
 
+def parse_darkhorse(input_f, low_lpi=0.0, high_lpi=0.6, return_genomes=False):
```
Comment (antgonza) [inline benchmark/parse_output.py:92]: Why use those defaults? I think it will be worth explaining.

**Comment by antgonza** [comment]
A few minor comments.

File: .travis.yml (hunk 1 above)
Comment (ekopylova) [inline .travis.yml]: Sorry not sure what you mean ..

File: benchmark/launch_benchmark.sh (hunk 3 above)
Comment (ekopylova) [inline benchmark/launch_benchmark.sh:35]: Shell interprets $10 as $1 unless there are brackets around ${10} [stackoverflow](http://stackoverflow.com/questions/4816635/how-do-i-access-arguments-to-functions-if-there-are-more-than-9-arguments)

File: benchmark/launch_benchmark.sh (hunk 4 above)
Comment (ekopylova) [inline benchmark/launch_benchmark.sh:46]: the init_command can have spaces in it, the "" contains the result into a string variable

File: benchmark/parse_output.py (hunk 5 above)
Comment (ekopylova) [inline benchmark/parse_output.py:92]: Good point, they are defaults in DarkHorse, will add

File: .travis.yml (hunk 1 above)
Comment (antgonza) [inline .travis.yml]: INSTALL -> Install, so it matches the rest of the comments.

File: benchmark/launch_benchmark.sh (hunk 3 above)
Comment (antgonza) [inline benchmark/launch_benchmark.sh:35]: K, thanks.

File: benchmark/launch_benchmark.sh (hunk 4 above)
Comment (antgonza) [inline benchmark/launch_benchmark.sh:46]: K, thanks.

**Comment by ekopylova** [comment]
@antgonza if everything looks good could you merge? Thanks!

**Pull request merged**